modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/LogStorageFactory.java

```diff
@@ -43,4 +47,16 @@ public interface LogStorageFactory extends ManuallyCloseable, LogSyncer {
      */
     @Override
     void close();
+
+    @Override
+    default CompletableFuture<Void> startAsync() {
+        start();
+        return nullCompletedFuture();
+    }
+
+    @Override
+    default CompletableFuture<Void> stopAsync() {
+        close();
+        return nullCompletedFuture();
+    }
```

Comment:
Is a good idea to provide default implementation there?

Reply:
I would propose to remove `start` and `close` methods completely, they are superseded by the similar methods from `IgniteComponent`

Reply:
Disregard my last comment, @sashapolo had already answered